runtime/colors.rs

```diff
@@ -8,12 +8,18 @@ use termcolor::{Ansi, ColorSpec, WriteColor};
 #[cfg(windows)]
 use termcolor::{BufferWriter, ColorChoice};
 
+// checks if the output is piped to a tty
+lazy_static::lazy_static! {
+  static ref IS_TTY: bool = atty::is(atty::Stream::Stdout);
+}
+
 lazy_static::lazy_static! {
   static ref NO_COLOR: bool = std::env::var_os("NO_COLOR").is_some();
 }
 
+// if the output is piped to a tty, use color
 pub fn use_color() -> bool {
-  !(*NO_COLOR)
+  !(*NO_COLOR) && *IS_TTY
```

Comment:
@bartlomieju there is already a function called `use_color` and hence I utilised it